src/main/java/net/glowstone/command/minecraft/GameModeCommand.java

```diff
@@ -54,38 +56,38 @@ public boolean execute(CommandSender sender, String label, String[] args) {
             for (Entity entity : matched) {
                 if (entity instanceof Player) {
                     Player player = (Player) entity;
-                    updateGameMode(sender, player, gamemode);
+                    updateGameMode(sender, player, gamemode, bundle);
                 }
             }
         } else {
             Player player = Bukkit.getPlayerExact(name);
             if (player == null) {
-                sender.sendMessage(ChatColor.RED + "Player '" + name + "' is not online.");
+                new LocalizedStringImpl("gamemode.offline", bundle)
+                        .sendInColor(ChatColor.RED, sender, name);
             } else {
-                updateGameMode(sender, player, gamemode);
+                updateGameMode(sender, player, gamemode, bundle);
             }
         }
         return true;
     }
 
-    private void updateGameMode(CommandSender sender, Player who, GameMode gameMode) {
-        String gameModeName = GameModeUtils.prettyPrint(gameMode);
+    private void updateGameMode(
+            CommandSender sender, Player who, GameMode gameMode, ResourceBundle bundle) {
+        String gameModeName = GameModeUtils.prettyPrint(gameMode, bundle.getLocale());
         who.setGameMode(gameMode);
-        if (!sender.equals(who)) {
-            sender.sendMessage(
-                who.getDisplayName() + "'s game mode has been updated to " + ChatColor.GRAY + ""
-                    + ChatColor.ITALIC + gameModeName + " Mode" + ChatColor.RESET);
+        if (sender.equals(who)) {
+            new LocalizedStringImpl("gamemode.done.self", bundle).send(sender, gameModeName);
+        } else {
+            new LocalizedStringImpl("gamemode.done", bundle)
```

Comment:
This changes the behavior to no longer notifying the player whose gamemode was set.

Reply:
Fixed.